packages/govern-create/tasks/govern.ts

```diff
@@ -0,0 +1,87 @@
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals,
+} from 'unique-names-generator'
+
+function buildName(name: string | null): string {
+  const uniqueName =
+    name ??
+    uniqueNamesGenerator({
+      dictionaries: [adjectives, colors, animals],
+      length: 2,
+      separator: '-',
+    })
+
+  return process.env.CD ? `github-${uniqueName}` : uniqueName
+}
+
+const format = (address: string, name: string, network: string): string =>
```

Comment:
Can we have one type of defining functions? I would prefer to use the normal function keyword for many reasons.